packages/platforms/accton/x86-64/as9716_32d/modules/builds/x86-64-accton-as9716-32d-ioport.c

```diff
@@ -0,0 +1,180 @@
+/*
+ * A platform driver for the Accton as9716 32d ioport
+ *
+ * Copyright (C) 2016 Accton Technology Corporation.
+ * Brandon Chuang <brandon_chuang@accton.com.tw>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+ */
+
+#include <linux/module.h>
+#include <linux/jiffies.h>
+#include <linux/hwmon.h>
+#include <linux/hwmon-sysfs.h>
+#include <linux/err.h>
+#include <linux/mutex.h>
+#include <linux/sysfs.h>
+#include <linux/slab.h>
+#include <linux/dmi.h>
+#include <linux/platform_device.h>
+#include <linux/delay.h>
+#include <linux/ioport.h>
+#include <asm/io.h>
+
+#define DRVNAME "as9716_32d_ioport"
+#define IOPORT_I2C_MUX_RST 0x50D
+
+static ssize_t show_i2c_mux_rst(struct device *dev, struct device_attribute *da,
+		char *buf);
+static ssize_t set_i2c_mux_rst(struct device *dev, struct device_attribute *da,
+		const char *buf, size_t count);
+
+static struct as9716_ioport_data *data = NULL;
+
+/* ioport data */
+struct as9716_ioport_data {
+	struct platform_device *pdev;
```

Comment:
Please format the file according to the Linux kernel coding style. There should be some tool like GNU indent or clang-format for that task.

Reply:
Update  x86-64-accton-as9716-32d-ioport.c formatted by GNU indent.

Reply:
Added SPDX header and formatted with GNU indent.